Makefile

```diff
@@ -1,6 +1,6 @@
 DISABLE_CACHE = 'false'
 
-ARCH = rpi# rpi/amd64/i386/armv7hf/armel
+ARCH = rpi# rpi/amd64/i386/armv7hf/armel/armhf-alpine/amd64-alpine/i386-alpine
```

Comment:
@nghiant2710 can we use the same architecture here and have the alpine be passed as a separate parameter?
e.g. we use `ARCH=rpi` and `BASE_DISTRO=-alpine`, and then:
- Use `${ARCH}${BASE_DISTRO}` to choose the Dockerfile
- keep using `ARCH` for the image name
- append `BASE_DISTRO` to the tag

So that we have images that look like `resin/amd64-supervisor:1.7.0-alpine`.

This would correspond to what we had discussed privately with @Page- and @dfunckt